### ISSUE TITLE
[ACI post-project connection] Create Cloud Connection Modal “Record a Run” state

### ISSUE THREAD

**Issue by marktnoonan** [cypress-io/cypress#23764]
**Context**

After logging in, users who have not connected their project are currently invited to do so and can complete connecting their project through what feels like the same modal that was present after login. When the project is connected, the modal is closed and the user is back in the app at whatever point they happened to be on.

**Requested Change**
Create a new state in the LoginConnect modal flow that shows the command needed to record a run and allows the user to copy the command to their clipboard. It should match this design, available in this [Figma board]()https://www.figma.com/file/V0lCq4uBplf9J3GZRiD02m/ACI-Fast-Follows?node-id=19661%3A172759)

![Screen Shot 2022-09-09 at 5 15 20 PM](https://user-images.githubusercontent.com/8340719/189447332-cc74ea1e-1b59-4723-a9e6-8f8a8b96f515.png)

For this ticket, just add the component but do not hook it up. That has been split out into a separate issue (#23766). The exact logic for when this should be shown will be exposed in #23765, but for the sake of doing things in parallel, there's no reason to wait on actually making the component.

**Acceptance Criteria**

- This new "Record Run" variant of the modal is displayed in a component test and has a percy snapshot
- It matches the design in Figma
- It has a Learn More Link: https://on.cypress.io/cypress-run-record-key
- The Learn Morn link uses the UTM params for where the flow was initially triggered from; same as used for login and other links in the Connect Dashboard Modal Flow; for example, if the flow was triggered from the nav, use medium="Nav" and campaign="Log In"
- Esc, click away, and the X in the top-right corner close the modal
- Component is not displayed in the app at all, and can be merged with no disruption to existing functionality.

**Suggested Implementation** (as a guide for estimates)

- Extract TerminalPrompt + recordCommand logic from RecordBanner.vue to get a reusable RecordPrompt component - this already matches the Figma design, no need to re-implement.
- Use the new RecordPrompt in RecordBanner 
- Use the new RecordPrompt in a new "RecordRunModal" that would be rendered by of CloudConnectModals.vue or LoginConnectModals.vue
- Add component tests convering all items in the AC

**Comment by baus** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Component-Testing-606e43bbf1b3b20013c142c1/issues/cypress-io/cypress/23764?planning-poker) with Zenhub @amehta265 @astone123 @lmiller1990 @marktnoonan @mike-plummer @rockindahizzy @warrensplayer @ZachJW34
